frontend/src/pages/profile/user.js

```diff
@@ -87,12 +80,12 @@ export class UserProfile extends React.Component {
     let tempUser = Object.assign({}, user, {name: tempName});
     this.saveUser(tempUser).then(response => {
       if (response !== undefined) {
-        this.showNotification('success', 'Name Updated', 'Your name has been updated.');
+        // this.showNotification('success', 'Name Updated', 'Your name has been updated.');
```

Comment:
We may want to remove this line entirely instead of commenting? Maybe launch a new toast?

Reply:
Let's call it early review state :smile: 

This will get replaced with `toast.success()` in this PR, but I'd love some feedback on the integration, styling, and patterns before using it here too.